src/main/java/mate/academy/springboot/datajpa/model/dto/ProductRequestDto.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.springboot.datajpa.model.dto;
+
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class ProductRequestDto {
+    private String title;
+    private BigDecimal price;
+    private String category;
```

Comment:
categoryId

Reply:
for what id?
request looks like this http://localhost:8080/products/search?categoryIn=PHONE
not http://localhost:8080/products/search?categoryIn=1

Reply:
it should be http://localhost:8080/products/search?categoryIn=1
Your API shouldn't depend on the name of a category, because the name can change, but id will remain the same